Gemfile.lock

```diff
@@ -209,7 +223,7 @@ DEPENDENCIES
   scim_rails!
   simplecov (< 0.18.0)
   simplecov_json_formatter
-  sqlite3 (~> 1.3.6)
+  sqlite3 (~> 1.3, < 1.5)
```

Comment:
are all of those GEfile.lock changes are intentional?

Reply:
Yes, I needed to this so I could get the project to run locally

Reply:
I can try rolling them back